README.md

```diff
@@ -42,13 +42,20 @@ docker pull docker.all-hands.dev/all-hands-ai/runtime:0.12-nikolaik
 
 docker run -it --pull=always \
     -e SANDBOX_RUNTIME_CONTAINER_IMAGE=docker.all-hands.dev/all-hands-ai/runtime:0.12-nikolaik \
+    -e SANDBOX_USER_ID=${SANDBOX_USER_ID:-1234} \
+    -e WORKSPACE_MOUNT_PATH=${WORKSPACE_BASE:-$PWD/workspace} \
```

Comment:
These were removed on purpose in the latest release. Is there a specific reason you have added them?

Reply:
problem: [file sync from container to local machine](https://openhands-ai.slack.com/archives/C06P5NCGSFP/p1730537599048529?thread_ts=1730482815.294839&cid=C06P5NCGSFP) 
add env variable to sync